examples/esp_idf/lightdb_stream/main/app_main.c

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "nvs.h"
+#include "shell.h"
+#include "wifi.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/semphr.h"
+#include <golioth/client.h>
+#include <golioth/stream.h>
+#include <string.h>
+
+#define TAG "lightdb_stream"
+
+struct golioth_client *client;
+static SemaphoreHandle_t _connected_sem = NULL;
+
+
+/**
+ * Definition taken from Zephyr SDK
```

Comment:
We don't need to match anything Zephyr specific here. The goal behind having examples for each platform (instead of shared examples) is to show how each service can be integrated into that platform, including any of that platform's specific features or weirdness. For this example, we can either just `printf` a float value directly, or even just use integer values (e.g. 20, 21, 22, etc.).